app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootsFragment.kt

```diff
@@ -56,13 +61,17 @@ internal class StorageRootsFragment : Fragment(), StorageRootClickedListener {
     ): View {
         val v: View = inflater.inflate(R.layout.fragment_storage_root, container, false)
 
-        titleView = v.findViewById(R.id.titleView)
+        suw_layout = v.findViewById(R.id.setup_wizard_layout)
         warningIcon = v.findViewById(R.id.warningIcon)
         warningText = v.findViewById(R.id.warningText)
         divider = v.findViewById(R.id.divider)
         listView = v.findViewById(R.id.listView)
         progressBar = v.findViewById(R.id.progressBar)
-        backView = v.findViewById(R.id.backView)
+        skipView = v.findViewById(R.id.skipView)
+
+        val icon: Drawable = suw_layout.getIcon()
+        icon.setTintList(getColorAccent(requireContext()))
+        suw_layout.setIcon(icon)
```

Comment:
It is too bad that `suw_layout` can't handle icon tinting internally. Kinda annoying having to set it everywhere.